src/components/orgs/CouponListPage/index.tsx

```diff
@@ -125,10 +121,18 @@ const CouponListPage: React.FC<ICouponListPageProps> = ({ supplierId }) => {
           supplier?.coupons.length > 0 ? (
             supplier?.coupons.map((coupon: ICoupon, key) => (
               <SupplierCoupons
-                icon={supplier.supplier?.id == session?.user.id ? Edit : Arrow}
+                couponTitle={coupon.title}
+                icon={supplier.supplier.id == session?.user.id ? Edit : Arrow}
```

Comment:
```suggestion
                icon={supplier.supplier.id === session?.user.id ? Edit : Arrow}
```

From now on, stop using simple comparison. **Always use `===` and `!==`**